src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/GC.Infrastructure.Core.csproj

```diff
@@ -23,5 +25,14 @@
   <ItemGroup>
     <ProjectReference Include="..\GC.Analysis.API\GC.Analysis.API.csproj" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(CustomTraceEvent)' == 'true'">
```

Comment:
The unit tests make no use of the dynamic events, why is this needed here?

Reply:
I don't understand fully either, but without directly referencing this, the project won't build because somewhere in the result comparison we accessed the properties of `GCStats` that was exposed on the API.

If the API perform the access and return raw values instead this wouldn't be necessary.